src/Whitelist/VulnerabilityExcluder.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { existsSync, readFileSync } from "fs";
```

Comment:
trivial: would we be able to use readFileSync with the `r+` system flag? https://nodejs.org/api/fs.html#fs_file_system_flags -- that might allow us to use just readFileSync and get rid of existsSync

Reply:
Done in: fd46cb8